src/display/svg.js

```diff
@@ -894,36 +896,41 @@ SVGGraphics = (function SVGGraphicsClosure() {
       current.setCurrentPoint(x, y);
     },
 
-    endPath: function SVGGraphics_endPath() {},
+    endPath: function SVGGraphics_endPath() {
+      if (this.pendingClip) {
```

Comment:
I think we should use this instead:

```javascript
if (!this.pendingClip) {
  return;
}
```

That way we avoid a level of indenting, which improves readability.